packages/transformer/src/IModelCloneContext.ts

```diff
@@ -6,30 +6,64 @@
  * @module iModels
  */
 import * as assert from "assert";
-import { DbResult, Id64, Id64String } from "@itwin/core-bentley";
+import { DbResult, Id64, Id64String, Logger } from "@itwin/core-bentley";
 import {
-  ConcreteEntityTypes, ElementAspectProps, EntityReference, IModelError,
-  PrimitiveTypeCode, RelatedElementProps,
+  Code,
+  CodeScopeSpec,
+  ConcreteEntityTypes, ElementAspectProps, ElementProps, EntityReference, IModel, IModelError,
+  PrimitiveTypeCode, PropertyMetaData, RelatedElement, RelatedElementProps,
 } from "@itwin/core-common";
 import {
-  ElementAspect, EntityReferences, IModelElementCloneContext, SQLiteDb,
+  Element, ElementAspect, EntityReferences, IModelElementCloneContext, IModelJsNative, SQLiteDb,
 } from "@itwin/core-backend";
 import { ECReferenceTypesCache } from "./ECReferenceTypesCache";
 import { EntityUnifier } from "./EntityUnifier";
+import { TransformerLoggerCategory } from "./TransformerLoggerCategory";
+
+const loggerCategory: string = TransformerLoggerCategory.IModelCloneContext;
 
 /** The context for transforming a *source* Element to a *target* Element and remapping internal identifiers to the target iModel.
  * @beta
  */
 export class IModelCloneContext extends IModelElementCloneContext {
 
   private _refTypesCache = new ECReferenceTypesCache();
+  private _aspectRemapTable = new Map<Id64String, Id64String>();
 
   /** perform necessary initialization to use a clone context, namely caching the reference types in the source's schemas */
   public override async initialize() {
     await this._refTypesCache.initAllSchemasInIModel(this.sourceDb);
   }
 
-  private _aspectRemapTable = new Map<Id64String, Id64String>();
+  /** Clone the specified source Element into ElementProps for the target iModel.
+   * @internal
+   */
+  public override cloneElement(sourceElement: Element, cloneOptions?: IModelJsNative.CloneElementOptions): ElementProps {
+    // eslint-disable-next-line @typescript-eslint/dot-notation
+    const targetElementProps: ElementProps = this["_nativeContext"].cloneElement(sourceElement.id, cloneOptions);
```

Comment:
you should make a PR to make `_nativeContext` protected so you won't have to do this.

Reply:
I told @KyryloVolotovskyi he can do this, because I will be refactoring usage of the native context in a near-future PR

Reply:
got it.